tests/CreateTestModelsTable.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+namespace SanderVanHooft\Invoicable;
```

Comment:
Can you try reverting this? Not sure, but it seems to trip up the Travis tests.

Reply:
Yes, this trips Travis up, but if I simply remove this, the style check fails.